src/styles/primeng-theme/utils/_theme-variables.scss

```diff
@@ -37,135 +37,71 @@ $button-height: $field-height;
 $button-disabled-color: $body-font-color;
 $button-radius: $field-border-radius;
 $button-inverted-color: $white;
+$button-hover-background-opacity: 0.12;
+
+$button-raised: (
```

Comment:
We not just have 2 main buttons, raised and flat and a "secondary" button that is icon and by default it's flat.